src/main/java/dev/carrico/controllers/StackController.java

```diff
@@ -44,4 +54,24 @@ public Boolean deleteBookById(@PathVariable int stackId){
         Boolean result = this.stackService.deleteStackById(stackId);
         return result;
     }
+
+    @PutMapping("/stacks/{stackId}/cards/{cardId}")
+    public Stack addCardToStack(@PathVariable int stackId, @PathVariable int cardId){
+        Stack stack = this.stackService.getStackById(stackId);
+        Card card = this.cardService.getCardById(cardId);
+        this.stackService.addCardToStack(card, stack);
+        return stack;
+    }
+
+    @PutMapping("/stacks/{stackId}/tags/{tagId}")
```

Comment:
Idk if I like this, 
Do we want to do something like 
`"/stacks/{stackId}?tag={tagId}"`